opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -34,12 +35,23 @@ const useStyles = makeStyles<Theme>((theme) => ({
     display: 'flex',
     alignItems: 'center',
   },
+  marginTop: {
+    marginTop: 20,
+  },
+  formContainer: {
+    margin: '20px 0',
+  },
+  representationContainer: {
+    marginTop: 20,
+    display: 'flex',
+  },
 }));
 
 const csvMapperValidation = (t: (s: string) => string) => Yup.object().shape({
   name: Yup.string().required(t('This field is required')),
   has_header: Yup.boolean().required(t('This field is required')),
   separator: Yup.string().required(t('This field is required')),
+  skipLineChar: Yup.string(),
```

Comment:
can we validate the char ? I guess it should be only one character or can it be more ? like this maybe : 
`Yup.string().max(1)`

Reply:
@nino-filigran @Jipegien, what do you think ? should we restrict to one character ?

Reply:
Ok I saw with @Jipegien we will restrict to one character. Thanks @SouadHadjiat.